js/modules/inputValidation.js

```diff
@@ -0,0 +1,114 @@
+const forms = document.querySelectorAll("form");
+let onSubmit = false;
```

Comment:
why do you need it?

check all the places

Reply:
Done.